### DISCUSSION TITLE
Updated Readme?

### DISCUSSION THREAD

**Discussion by diapermodder** [zippity21/Stardew_Valley_Regression_Mod#130] [General]
Hi, I've followed the instructions on the readme. Did not work. Then, I tried installing using earlier (pre-1.6) versions, also did not work. I would love to play this mod, it just seems like there's no possible way to get it to work anymore? Anyone had any success recently either before or after 1.6? Thank you!

**Comment by Jawsh44** [comment]
I'm not sure if you can get the mod to work with the latest version of Stardew, but I've got it working on version 1.6.8

ContentPatcher 2.0.6
Smapi 4.0.8
And PyTk is no longer required if you read the latest release
https://github.com/zippity21/Stardew_Valley_Regression_Mod/releases/tag/v1.3.0

It works pretty well, the only issue I'm having is that sometimes notifications don't show up properly, not a huge deal but can be a bit inconvenient sometimes
Hoping we get an update sometime soon though 😅
